person.rb

```diff
@@ -4,10 +4,10 @@
 require_relative 'trimmer_decorator'
 require_relative 'rental'
 class Person < Nameable
-  attr_accessor :name, :age
-  attr_reader :id, :rentals
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
 
-  def initialize(age, name = 'Unknown', parent_permission: true)
+  def initialize(age:, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] As explained above, please kindly fix this by using the  below code snippet
- code snippet
```ruby
def initialize(age, name = 'Unknown', parent_permission: true)
```